core/server/master/src/test/java/alluxio/master/AlluxioMasterProcessTest.java

```diff
@@ -163,6 +202,7 @@ public void stopAfterStandbyTransition() throws Exception {
     waitForServing(ServiceType.MASTER_WEB);
     assertTrue(isBound(mRpcPort));
     assertTrue(isBound(mWebPort));
+    Thread.sleep(2000);
```

Comment:
Clock-based sleep is flakey especially in github actions. Is this needed?

Reply:
This test was a flaky test, I add this sleep to pass this test for now. I'm not good at to fix this UT, so just add a sleep here, I can revert it if you think this is not a good practice.

Reply:
I see. Let's revert this line because the change is irrelevant to this PR. We will fix the flaky test somewhere else. Thx for the effort though!

Reply:
OK.